src/ansys/units/dimensions.py

```diff
@@ -1,142 +1,99 @@
 """Provides the ``Dimension`` class."""
```

Comment:
For the docstring, maybe something more comprehensive like:

```Provides the implementation of dimensions for the units library.

The valid fundamental base dimensions are defined by the `Enum` class called `BaseDimensions`.

The `Dimensions` provide for defining a dimensional quantity from it's `BaseDimensions` on which mathematical operations can be performed to facilitate building other dimensions.
``` 
Does not need to be identical to that, just an idea.

Reply:
Is this for the `Dimensions` or `BaseDimensions` docstring?

Reply:
Yeah, I meant `Dimensions`. I suppose that the `BaseDimensions` enum defines the valid base quantities right but a `Dimension` is providing more than that.